src/Components/OperationsGroup.jsx

```diff
@@ -88,5 +89,9 @@ const useStyles = makeStyles({
     paddingBottom: '70px',
     minHeight: '150px',
   },
+  notesButtonContainer: {
+    border: (props) => props.isCommentsOn ? 'none' : '1px solid lime',
```

Comment:
Is this the intended behavior?  Thought I commented on this but I don't see my review anymore.

I was expecting the highlight to show up when comments are active, but be off when they're not.  This is inverted.  Happy to discuss.